### PR TITLE
docs: Fix snowflake article

### PR DESCRIPTION
**Pull request by adela-bytebase**
It was broken ...

### CODE REVIEW

**Review by tianzhou — CHANGES_REQUESTED** [review]
You need to sync with the head

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/blog/database-change-management-with-snowflake.md [hunk 1]
```diff
@@ -110,6 +111,14 @@ In Step 4, you actually created an issue in UI workflow and then executed it. Le
...
 
 3. Input title, SQL, and Assignee, and click **Create**.
+````sql
+CREATE SCHEMA DEMO_UI;
+CREATE TABLE HELLO_WORLD
+(
+  FIRST_NAME VARCHAR
+  ,LAST_NAME VARCHAR
+);
+````
```
[the remark this answers is not in the document]
Comment (tianzhou) [reply content/blog/database-change-management-with-snowflake.md:121]: I guess we copied from 
CleanShot 2022-12-27 at 21 23 08


But that's not the typical writing.

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/blog/database-change-management-with-snowflake.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adela-bytebase) [reply content/blog/database-change-management-with-snowflake.md:121]: yes ...

File: content/blog/database-change-management-with-snowflake.md [hunk 2]
```diff
@@ -110,6 +111,13 @@ In Step 4, you actually created an issue in UI workflow and then executed it. Le
 ![bb-alter-schema-test](/static/blog/database-change-management-with-snowflake/bb-alter-schema-test.webp)
 
 3. Input title, SQL, and Assignee, and click **Create**.
+````sql
+CREATE SCHEMA DEMO_UI;
+CREATE TABLE HELLO_WORLD
+(
+  FIRST_NAME VARCHAR,
+  LAST_NAME VARCHAR
+);
```
Comment (tianzhou) [inline content/blog/database-change-management-with-snowflake.md:120]:
```suggestion
````sql
CREATE SCHEMA DEMO_UI;
CREATE TABLE HELLO_WORLD
(
  FIRST_NAME VARCHAR,
  LAST_NAME VARCHAR
);
````
```

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/blog/database-change-management-with-snowflake.md (hunk 2 above)
Comment (tianzhou) [reply content/blog/database-change-management-with-snowflake.md:120]: Need to have trailing ````

**Pull request merged**